test/fixtures/animaldb_corrupted.json

```diff
@@ -1,3 +1,3 @@
-{"name":"@cloudant/couchbackup","version":"2.9.10","mode":"full"}
+{"name":"@cloudant/couchbackup","version":"2.9.10","mode":"full","attachments":false}
```

Comment:
Do we have/need an e2e test with a file without the `attachments` field?
I see there is one unit test for that backward compatible case in `test/restoreMappings.js`, but we would have bigger coverage by leaving one of these files as-is.

Reply:
The `_old` files specifically test restore of backup files that predate the metadata header. So they cover a no metadata file case.
We can't leave the `expected` files as-is because they are used to compare an actual backup during the tests and now the code always writes the `attachments` flag in the metadata header.

This one file though I think can be used to cover a restore of a backup file with a metadata header, but no attachments flag - fixed in 5ee653d39f52d7cc2693cdca3f8a0b6d438b98a3.